pkg/nsx/services/inventory/cluster.go

```diff
@@ -2,17 +2,44 @@ package inventory
 
 import (
 	"context"
+	"fmt"
+
+	"net/http"
 
 	"github.com/vmware/go-vmware-nsxt/containerinventory"
 
+	nsx_util "github.com/vmware-tanzu/nsx-operator/pkg/nsx/util"
 	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
-func (s *InventoryService) GetContainerCluster() (containerinventory.ContainerCluster, error) {
+const (
+	baseUrl = "api/v1/fabric/container-clusters/%s"
+)
+
+func (s *InventoryService) GetContainerCluster(cleanup bool) (containerinventory.ContainerCluster, error) {
 	clusterUUID := util.GetClusterUUID(s.NSXConfig.Cluster).String()
 	log.Info("Send request to NSX to get inventory cluster", "Cluster id", clusterUUID)
```

Comment:
log.Info("Send request to NSX to get inventory cluster", "ClusterUID", clusterUUID)